docs/book/src/plugins/grafana-v1-alpha.md

```diff
@@ -77,8 +77,20 @@ See an example of how to use the plugin in your project:
 - Description:
   - Per-second rate of total reconciliation as measured over the last 5 minutes
   - Per-second rate of reconciliation errors as measured over the last 5 minutes
+- Sample: <img width="1430" src="https://user-images.githubusercontent.com/18136486/176122555-f3493658-6c99-4ad6-a9b7-63d85620d370.png">
 
-<img width="1430" alt="Screen Shot 2022-06-28 at 3 43 10 AM" src="https://user-images.githubusercontent.com/18136486/176122555-f3493658-6c99-4ad6-a9b7-63d85620d370.png">
+#### Controller Resources Usage (CPU, Memory)
+
+- Metrics:
+  - process_cpu_seconds_total
+  - process_resident_memory_bytes
+- Query:
+  - rate(process_cpu_seconds_total{job="$job", namespace="$namespace", pod="$pod"}[5m]) \* 100
+  - process_resident_memory_bytes{job="$job", namespace="$namespace", pod="$pod"}
```

Comment:
but if we are looking for any pod that is not the manager one right?
WDYT about we looking for the label : https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v3/config/manager/manager.yaml#L5

Reply:
> but if we are looking for any pod that is not the manager one right?

- Since this `process_resident_memory_bytes` metrics [roots from controller-runtime](https://github.com/kubernetes-sigs/controller-runtime/blob/v0.12.2/pkg/internal/controller/metrics/metrics.go#L74), I think it is for controller-manager.
- What else pods that the `cpu/memory` panels may cover?

> WDYT about we looking for the label?

- Are you suggesting that we compose the query to select cpu/memory metrics based on `K8s labels`?
    Guess it may be hard, because the prometheus metrics by default may not contain the info of `K8s labels` right?
    For example, this `process_resident_memory_bytes`'s own labels do not cover `k8s labels` for the controller:
   <img width="1440" alt="Screen Shot 2022-07-05 at 12 58 25 PM" src="https://user-images.githubusercontent.com/18136486/177381439-31c40b62-546f-45ef-a680-27a8e1844a11.png">

Reply:
Am I missing the point that we are also expecting to monitor the status of the pods that are **launched by the operator/manager**?
(For example: the pod of a job that is scheduled by a customized cron-job operator?)

Reply:
@Kavinjsir , 

But we can use the container name in this way. What do you think about we filter by container=manager?

Reply:
@camilamacedo86 @Kavinjsir  The K8s labels are configurable and can be changed by the user, so that would come under custom metrics. If `container=manager` label is something which kb scaffolding [adds](https://github.com/kubernetes-sigs/kubebuilder/blob/c329411c85ce9df7ae20156713bca8b9f01fd9cb/testdata/project-v3/config/manager/manager.yaml#L5) it, then its pretty easy to change it and error out since we hardcode it. What do you think about using labels but letting the user to pass them? If not by maybe parsing the deployment manifest where manager is defined.